.github/workflows/build.yml

```diff
@@ -65,9 +65,9 @@ jobs:
 
   # There is a single job in this workflow. It's configured to run on the latest available version of Ubuntu.
   build-and-push-image:
-    needs:
-      - lint
-      - format-check
+    # needs:
```

Comment:
NOTE: DO NOT ALLOW Merge without uncommenting the needs bit here (and the other commands in this doc)

Reply:
(fixed once I got it working - thanks Jason for responding to me late!)